tests/network/kubemacpool/explicit_range/conftest.py

```
@@ -0,0 +1,82 @@
+            }
+        },
+        list_resource_reconcile=[NetworkAddonsConfig],
+        wait_for_reconcile_post_update=True,
+        admin_client=admin_client,
+    ):
+        yield
+
+
+@pytest.fixture()
+def custom_range_hco_mac_pool(
+    admin_client: DynamicClient,
```

Ruff (0.14.10)

Unused function argument: `admin_client`

(ARG001)